nix/grpc.nix

```diff
@@ -1,42 +1,45 @@
-{ darwin, stdenv, lib, fetchgit, fixDarwinDylibNames, autoconf, automake, libtool, which, zlib
-, openssl
-}:
+{ stdenv, fetchFromGitHub, cmake, zlib, c-ares, pkgconfig, openssl, protobuf, gflags }:
```

Comment:
The nix file is just taken from `nixpkgs` but I think it makes sense to inline it here so we don’t need to tie the nixpkgs rev to the gRPC version.

Reply:
I think it would still be better to get it from the pinned `nixpkgs` because if we want to bump the version we can easily do so with an override in the `release.nix`, reducing the amount of Nix code we need to maintain.